manticore/core/manticore.py

```diff
@@ -1219,7 +1223,28 @@ def save_run_data(self):
         with self._output.save_stream("manticore.yml") as f:
             config.save(f)
 
+        with self._output.save_stream("global.solver_stats") as f:
+            for s, n in sorted(SOLVER_STATS.items()):
+                f.write("%s: %d\n" % (s, n))
+
+        if SOLVER_STATS["timeout"] > 0 or SOLVER_STATS["unknown"] > 0:
+            logger.info(
+                "Warning: the SMT solvers returned timeout or unknown for certain program paths. Results could not cover the entire set of possible paths"
+            )
+
         logger.info("Results in %s", self._output.store.uri)
+
+        time_ended = time.time()
+
+        with self.locked_context() as context:
+            if "time_started" in context:
+                time_elapsed = time_ended - context["time_started"]
+                logger.info("Total time: %s", time_elapsed)
+                context["time_ended"] = time_ended
+                context["time_elapsed"] = time_elapsed
+            else:
+                logger.info("Warning: manticore failed to run")
```

Comment:
* See above re: info/warning disconnect
* "Manticore failed to run" seems a little bit aggressive. Maybe "Run start time not found in context. Was `save_run_data` called without calling `run`?"

Reply:
The log level for this was changed to `warning`. This error will happen if you provide manticore with incorrect binary code or bytecode. Typically, when a smart contract is not properly compiled. Right now, this is output:

```
2021-04-06 09:15:54,463: [13997] m.c.manticore:INFO: Generated testcase No. 0 - NO STATE RESULT (?)(0 txs)
2021-04-06 09:15:54,464: [13997] m.c.plugin:WARNING: Caught will_solve in state None, but failed to capture its initialization
2021-04-06 09:15:54,475: [13984] m.c.manticore:INFO: Results in /home/g/Data/manticore-benchmarks/mcore_mkg0pep9
```

Here manticore does not provide any good feedback that it failed to run beyond the "NO STATE RESULT (?)", that's why "Manticore failed to run" can be appropriated (but I'm open to suggestions).